tooling/commands/ci.js

```diff
@@ -45,7 +46,7 @@ module.exports = {
   },
   handler: wrapHandler(async (argv) => {
     let packages = argv.integration ?
-      await updated({dependents: true, npm: true}) :
+      await list() :
```

Comment:
@adamweeks
Grabbed `list()` from here
https://github.com/webex/webex-js-sdk/blob/master/tooling/commands/list.js#L26
And then added filtering is done in the `ci.js` file

Reply:
If the `list` command is just using `updated`, we can keep using `updated` in `ci.js`. I thought you said it was a custom thing you wrote.